tox-update.ini

```diff
@@ -75,13 +75,14 @@ commands=
   ansible-playbook -vv -i {changedir}/{env:INVENTORY} {toxinidir}/rolling_update.yml --extra-vars "\
       ireallymeanit=yes \
       fetch_directory={env:FETCH_DIRECTORY:{changedir}/fetch} \
+      ceph_stable_release={env:UPDATE_CEPH_STABLE_RELEASE:octopus} \
```

Comment:
```suggestion
```

Reply:
seems to be mandatory for non container update

Reply:
not here since we are using `dev_setup.yml`